introduction.md

```diff
@@ -1 +1,5 @@
+*Git, 
+*JavaScript, 
+*Ruby
+
 https://www.codeschool.com/paths/javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?